db/migrate/20151116114008_retire_links_column_from_links_sets.rb

```diff
@@ -0,0 +1,5 @@
+class RetireLinksColumnFromLinksSets < ActiveRecord::Migration
+  def change
+    rename_column :link_sets, :links, :legacy_links
+  end
+end
```

Comment:
We normally leave the existing fields as is. We then add a story to trello to remove redundant fields.

Therefore, I'd leave this migration out for now (if possible).